frontend/public/locales/en-US/analytics.json

```diff
@@ -461,6 +461,29 @@
       "segmentDescPlaceholder": "Provide a short description",
       "filterByTimeRange": "Filter by a date and time range"
     },
+    "details": {
+      "title": "Segment Details",
+      "desc": "Segment summary help you view the key information of a segment and monitor the segment trends.",
+      "date": "Date",
+      "userNumber": "User number",
+      "segmentUserNumber": "Segment user number",
+      "totalUserNumber": "Total user number",
+      "percentageOfTotalUser": "% of total user",
+      "noSegmentJobData": "No segment job data available",
+      "refreshSchedule": "Refresh Schedule",
+      "refreshExpiration": "Auto Refresh Expiration",
+      "createdBy": "Created By",
+      "updatedBy": "Updated By",
+      "segmentHistory": "Segment History",
+      "segmentHistoryDesc": "Latest 30 segment refresh history",
+      "startTime": "Job start time",
+      "endTime": "Job end time",
+      "jobStatus": "Job status",
+      "export": "Export",
+      "segmentUserSample": "User Samples",
+      "segmentUserSampleDesc": "User samples from latest completed segment refresh",
```

Comment:
I am confused.

Reply:
We will read up to 50 users info from user segment as sample data. The user info contains id, first traffic properties, custom properties etc.